docs/api/en/extras/PMREMGenerator.html

```diff
@@ -15,7 +15,36 @@ <h1>[name]</h1>
 			CubeUV format that allows us to perform custom interpolation so that we can support nonlinear formats such as RGBE.
```

Comment:
Does the RGBE comment still apply?

Reply:
No, good catch, since it's all float now.

Reply:
+1